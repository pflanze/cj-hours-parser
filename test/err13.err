Error: can't process file "test/err13.txt"

Caused by:
    0: can't parse line 2, " ["
    1: can't parse as a processing instruction: " ["
    2: column 1: expecting an identifier
    3: does not start with an ascii alphabetic character
