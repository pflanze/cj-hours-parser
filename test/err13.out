item=SingleLineComment(Owned(" this is a comment"))
    =SingleLineComment(Owned(" this is a comment"))
